test/helpers/foundry/types.ts

```diff
@@ -54,7 +54,12 @@ export enum Binary {
 /**
  * Tuple representing all members of the {@link Binary} enum.
  */
-export type BinariesTuple = UnionToTuple<Binary>;
+export type BinariesTuple = [
+  Binary.Anvil,
+  Binary.Forge,
+  Binary.Cast,
+  Binary.Chisel,
+];
```

Comment:
Why did this need to be hardcoded?

Reply:
whoa nice catch! I thought it was failing lint but it's not. I've changed it back here :pray: